lib/wickedbase.pm

```diff
@@ -234,6 +234,31 @@ sub get_test_result {
     }
 }
 
+sub get_wicked_logs {
```

Comment:
so let's start adding functions in this file with some documentation the same way as it is done in utils.pm

Reply:
I plan to add all the documentation in another PR if you don't mind.

Reply:
ok, make sense